specification/dsdl/attributes.tex

```diff
@@ -141,7 +140,7 @@ \subsection{Intrinsic attributes}
 \subsubsection{Offset attribute}
 
 The offset attribute is referred to by its identifier ``\verb|_offset_|''.
-Its value is of type $\texttt{set}_\texttt{<rational>}$.
+Its value is of type $\texttt{set}_\texttt{ <rational>}$.
```

Comment:
Why the space? This is what it looks like currently, it seems correct:

![image](https://user-images.githubusercontent.com/3298404/67310647-d691d080-f506-11e9-8adc-6a76939cbc45.png)

Reply:
Sorry, not familiar with this. I'll revert.